elements/pfe-navigation/src/_shared-assets.scss

```diff
@@ -31,11 +31,19 @@ $USE-FALLBACK: true; // turn on automatic fallbacks for broadcasted variables
 
 %trigger {
     font-family: pfe-var(font-family);
-    font-size:   pfe-local($cssvar: FontSize, $region: trigger, $fallback: #{$trigger--FontSize--fallback});
+    font-size:   pfe-local($cssvar: FontSize, $region: trigger, $fallback: #{$trigger--FontSize--fallback}) !important;
```

Comment:
the font-size fallback is set to grab the theme font size "zeta" which is 14px, but we are using 16px which is "epsilon".

Reply:
@starryeyez024 set this to use the header--epsilon font size.